documentation/modules/deploying/proc-deploy-kafka-cluster.adoc

```diff
@@ -10,7 +10,7 @@ This procedure shows how to deploy a Kafka cluster to your Kubernetes cluster us
 
 The deployment uses a YAML file to provide the specification to create a `Kafka` resource.
 
-Strimzi provides the following xref:config-examples-{context}[example files] you can use to create a Kafka cluster:
+Strimzi provides the following xref:config-examples-{context}[example files] to create a Kafka cluster that uses ZooKeeper for cluster management:
```

Comment:
The example files have KRaft-based examples as well. You do not use them in this section and they are not specifically listed. But I wonder if leaving it more generic is better. Or maybe at least mention that KRaft examples are in `examples/kafka/nodepools`?

Reply:
The node pools examples are mentioned in a separate deploying node pools section after this one: [5.3.2. (Preview) Deploying Kafka node pools](https://strimzi.io/docs/operators/in-development/deploying#deploying-kafka-node-pools-str)

We should take the `preview` off the node pool sections. I'll do that in a separate PR.